backend/app/app/templates/profile.html

```diff
@@ -53,6 +53,83 @@
         });
     </script>
 
+
+      <script>
+
+          var app = angular.module("app", ["myDirectives"]);
+          var myDirectives = angular.module("myDirectives", []);
+          var boolCheck = 0;
+          app.controller("UploadController", function ($log) {
+            this.upload = function () {
+              $log.info("Uploading:", this.file || "no file selected!");
+            }
+          })
+
+
+          myDirectives.directive("myFileUpload", function ($compile) {
+            return {
+              restrict: "AE",
+              require: "ngModel",
+              scope: true,
+              link: link
+          };
+
+          function link (scope, element, attrs, ngModel) {
+          var input = angular.element("<input type=\"file\" style=\"display: none;\">");
+
+          input.bind("browse", function () {
+            this.click();
+          });
+
+          input.bind("change", function (changed) {
+            const file = this.files[0];
+            const  fileType = file['type'];
+            const validImageTypes = ['image/gif', 'image/jpeg', 'image/png'];
+            if (!validImageTypes.includes(fileType)) {
+                return window.alert("Wrong format")
```

Comment:
replace alert with toastr.error() and make the message more specific. Maybe something like "Please upload an image".